kotlin/workflow-ui-android/build.gradle

```diff
@@ -27,6 +27,10 @@ android.packagingOptions {
   exclude 'META-INF/atomicfu.kotlin_module'
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+  kotlinOptions.freeCompilerArgs += ["-Xuse-experimental=kotlin.Experimental"]
+}
+
```

Comment:
Why `tasks.withType(…)` instead of `compileKotlin`?

```groovy
compileKotlin {
    kotlinOptions {
        freeCompilerArgs += "-Xuse-experimental=org.mylibrary.ExperimentalMarker"
    }
}
```
_from https://kotlinlang.org/docs/reference/experimental.html#module-wide-use_

Reply:
Also shouldn't we only need to allowlist our own `ExperimentalWorkflowUi` annotation?

Reply:
Didn't see that recipe in my five seconds of Googling, will fix. 

WRT why, compiler warnings say that you have to opt into the use of the @Experimental annotation itself. 

And I actually don't want to whitelist _our_ annotation, b/c the warnings it causes are useful to ensure we've put the annotation in enough places.

Reply:
https://stackoverflow.com/questions/44141076/compilekotlin-block-in-build-gradle-file-throws-error-could-not-find-method-com

Reply:
🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️